api/bitcoin/no-features.txt

```diff
@@ -9186,7 +9186,20 @@ pub use bitcoin::locktime::absolute::Time
 pub use bitcoin::locktime::relative::Height
 pub use bitcoin::locktime::relative::Time
 pub use bitcoin::locktime::relative::TimeOverflowError
+pub use bitcoin::parse::ContainsPrefixError
+pub use bitcoin::parse::MissingPrefixError
 pub use bitcoin::parse::ParseIntError
+pub use bitcoin::parse::PrefixedHexError
+pub use bitcoin::parse::UnprefixedHexError
+pub use bitcoin::parse::hex_check_unprefixed
+pub use bitcoin::parse::hex_remove_prefix
+pub use bitcoin::parse::hex_u128
+pub use bitcoin::parse::hex_u128_unchecked
+pub use bitcoin::parse::hex_u128_unprefixed
+pub use bitcoin::parse::hex_u32
+pub use bitcoin::parse::hex_u32_unchecked
+pub use bitcoin::parse::hex_u32_unprefixed
+pub use bitcoin::parse::int
```

Comment:
I believe API changes should not be mixed with the rest.

Reply:
just split it into 2 commits